homeassistant/components/epson/media_player.py

```diff
@@ -59,17 +59,20 @@
     {
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_PORT, default=80): cv.port,
     }
 )
 
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up the Epson projector from a config entry."""
-    unique_id = config_entry.entry_id
```

Comment:
I didn't realize we had already set a unique id using the config entry entry_id. Then we'd need to migrate all the existing entity entries to use the new unique_id. There is a helper to migrate entity entries in the entity registry.

Reply:
We want to migrate before loading the platform.